runtime/onert/core/src/ir/OperationValidator.cc

```diff
@@ -451,6 +451,19 @@ void OperationValidator::visit(const operation::Reverse &node)
   OP_REQUIRES(isSameType(output_index, input_index));
 }
 
+void OperationValidator::visit(const operation::RoPE &node)
+{
+  const auto input_index{node.getInputs().at(operation::RoPE::Input::INPUT)};
+  const auto sin_index{node.getInputs().at(operation::RoPE::Input::SIN_TABLE)};
+  const auto cos_index{node.getInputs().at(operation::RoPE::Input::COS_TABLE)};
```

Comment:
As I mentioned before, I don't think `sin` and `cos` tables are mandatory inputs. It is just one possible implementation.

But if we decide to implement and support sin/cos tables only for a while. I am okay.

cc @hseok-oh, @chunseoklee Are you okay in this approach?

Reply:
@glistening 
i agree your opinion.
so i hope i keep the current implementation and i will consider generating sin/cos tables by looking at the implementation of toolchain and onert upper layers in the future.
thanks